### ISSUE TITLE
Cobwebs layer under catwalk tiles

### ISSUE THREAD

**Issue by Sadboysuss** [tgstation/tgstation#87022]
### Issue Summary:

![image](https://github.com/user-attachments/assets/13906102-39d3-4938-bdb8-827349010c88)

### Reproduction Steps:

place cobweb on a catwalk tile

**Comment by 00-Steven** [comment]
yeah this is probably due to a mix of the floor plane flattening and my changes, think it'd be best fixed by moving `ABOVE_OPEN_TURF_LAYER` back above catwalks and creating a new layer like iunno `BELOW_CATWALK_LAYER` and splitting what currently uses `ABOVE_OPEN_TURF_LAYER` into those two categories
